my settings
Index: dwm/config.h
===================================================================
--- dwm.orig/config.h
+++ dwm/config.h
@@ -114,3 +114,11 @@ static const Button buttons[] = {
 	{ ClkTagBar,            MODKEY,         Button3,        toggletag,      {0} },
 };
 
+/* autostart */
+/* https://dwm.suckless.org/patches/cool_autostart/ */
+static const char *const autostart[] = {
+	"sh", "-c", "~/.fehbg", NULL,
+	"picom", NULL,
+	NULL
+};
+
